NeuroID/NeuroIDClass/Extensions/NIDAdvancedDevice.swift

```diff
@@ -16,11 +16,11 @@ public extension NeuroID {
             return started
         }
        
+        NeuroID.isAdvancedDevice = advancedDeviceSignals
+        
         if advancedDeviceSignals {
             // call stored value, if expired then clear and get new one, else send existing
-            if !getCachedADV() {
-                getNewADV()
-            }
+            captureAdvancedDevice()
```

Comment:
The if statement above is redundant because this method checks if ADV is allowed